app/jenkins-for-jira-ui/src/components/MainPage/TopPanel/TopPanel.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { cx } from '@emotion/css';
+import {
+	topPanelContainer,
+	topPanelContent,
+	topPanelContentHeader,
+	topPanelContentHeaderContainer,
+	topPanelImgContainer,
+	topPanelImg
+} from '../MainPage.styles';
+import { PluginIcon } from '../../icons/PluginIcon';
+
+const TopPanel = (): JSX.Element => {
+	return (
+		<div className={cx(topPanelContainer)}>
+			<div className={cx(topPanelContentHeaderContainer)}>
+				<h2 className={cx(topPanelContentHeader)}>Server management</h2>
+				<p className={cx(topPanelContent)}>Jenkins for Jira lets your teams <u>keep track of code
+					they build and deploy</u> on Jenkins servers.</p>
+				{/* TODO - add link */}
+				<p className={cx(topPanelContent)}>Follow the <strong>set up guide</strong> for each server to
```

Comment:
Clicky stuff falls into the functionality bucket so will circle back on this later.